docs/content/functional/ZRC/documentation.md

```diff
@@ -0,0 +1,17 @@
+#Functionele Documentatie ZAPI ZaakRegistratieComponent (ZRC)
```

Comment:
@michielverhoef kan je dit in een aparte branch en pull request doen? Deze PR gaat over de DevOps inrichting

Reply:
Ik heb dit volgens mij in een aparte PR gedaan, tenminste, dat was wel de bedoeling... Sorry!

Reply:
Hoe verwijder ik dit weer uit deze pull request?

Reply:
doe ik morgen even, komt goed :+1: